src/jobs/newsletters.ts

```diff
@@ -98,13 +103,58 @@ const rowToNewsletter = ({
 			brazeSubscribeAttributeNameAlternate
 				?.split(',')
 				?.map((a) => a.trim()),
-	} as EmailNewsletter);
+	};
+	const decodedNewsletter = BaseNewsletterCodec.decode(newsletter);
+	if (isRight(decodedNewsletter)) {
+		return decodedNewsletter.right;
+	} else {
+		console.log(`Could not decode newsletter: ${newsletter}`);
```

Comment:
```suggestion
		console.log(`Could not decode newsletter: ${newsletter}`);
		console.log(
			decodedNewsletter.left
				.map((validationError) => validationError.message)
				.join(', '),
		);
```
If we wanted to get more context on why the validation failed, we could log the errors from the type library.